<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAppSetting</class>
 <widget class="QWidget" name="WidgetAppSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetSettingBase" native="true">
     <property name="background" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>24</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>确认</string>
          </property>
          <property name="Ok" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="text">
           <string>重置</string>
          </property>
          <property name="Ok" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widgetSettingItems" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="topMargin">
             <number>18</number>
            </property>
            <property name="verticalSpacing">
             <number>24</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>编码设置</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>编码类型：</string>
                </property>
                <property name="Mark" stdset="0">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbbCodeType"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>310</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="topMargin">
             <number>18</number>
            </property>
            <property name="verticalSpacing">
             <number>24</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>网络设置</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>默认网卡：</string>
                </property>
                <property name="Mark" stdset="0">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbbNetworkCard"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
